databricks-sdk-java/src/main/java/com/databricks/sdk/core/BodyLogger.java

```diff
@@ -48,8 +48,11 @@ public String redactedDump(String body) {
       Object result = recursiveMarshal(rootNode, maxBytes);
       return mapper.writerWithDefaultPrettyPrinter().writeValueAsString(result);
     } catch (JsonProcessingException e) {
-      // Unable to unmarshal means the body isn't JSON (or something else)
-      return String.format("[unable to marshal: %s]", e.getMessage());
+      // Unable to unmarshal means the body isn't JSON
+      if (body.length() > maxBytes) {
+        return body.substring(0, maxBytes - 3) + "...";
```

Comment:
This value should be escaped, seeing as it ends up in log files. If we log a files API response it can be any type of binary data and it can mess up your terminal.

Reply:
For now, I think this is safe-ish. The main value that triggers this case is "<InputStream>", since that is what we set debugBody to when streaming a response. Otherwise, it behaves the same as existing logging. I'll change this impl to reuse onlyNBytes, and we can pick this up as a follow-up taks.

Reply:
Also for future reference: the implementation currently splits the debug body string by newlines and prepend the `> ` prefix, so it should not be possible for attackers to trigger printing out specific log files that don't have that prefix.